examples/cargo_without_cmake/src/main.rs

```diff
@@ -28,6 +28,13 @@ fn main() {
         engine.load(&QUrl::from("qrc:/qt/qml/com/kdab/cxx_qt/demo/qml/main.qml"));
     }
 
+    if let Some(engine) = engine.as_mut() {
```

Comment:
do we want this in the actual examples?

Reply:
Doesn't hurt anything I guess and is nice to show off.